production/tanka/grafana-agent/smoke/main.libsonnet

```diff
@@ -0,0 +1,15 @@
+local smoke = import './internal/smoke.libsonnet';
+
+{
+    _images:: {
+        agentsmoke: 'us.gcr.io/kubernetes-dev/grafana/agent-smoke:latest',
+    },
+
+    new(name='grafana-agent-smoke', namespace='grafana-agent-smoke', image=self._images.agentsmoke):: {
+        local this = self,
```

Comment:
probably don't need this, honestly this whole file seems a bit unnecessary if I just moved smoke.libsonnet to main.libsonnet